server/utils/url.ts

```diff
@@ -0,0 +1,5 @@
+import randomstring from "randomstring";
+
+const DocumentUrlIdLen = 10;
+
+export const generateDocUrlId = () => randomstring.generate(DocumentUrlIdLen);
```

Comment:
```suggestion
const UrlIdLength = 10;

export const generateUrlId = () => randomstring.generate(UrlIdLength);
```

Lets just make it generic, the same method exists for collection – so could use this there too